roles/containerd/defaults/main.yml

```diff
@@ -18,8 +18,54 @@ containerd_version: "{{ containerd_versions | last }}"
 containerd_archive_checksums:
   amd64:
     1.7.17: 04cf937349f82d29fe98553ff45a7e9ea2ed6b81fe6514e3679cf263b50409ff
+    1.7.18: a24b05b341c155a0ec367d3d0fd1d437c09a0261dffdecc0e44e9abbf2d02aca
+    1.7.19: 97f75e60f0ad19d335b1d23385835df721cad4492740d50576997f2717dc3f94
+    1.7.20: e09410787b6f392748959177a84e024424f75d7aff33ea1c5b783f2260edce67
+    1.7.21: 3d1fcdfd0b141f4dc4916b7aee7f9a7773dc344baffc8954e1ca66b1adc5c120
+    1.7.22: f8b2d935d1f86003f4e0c1af3b9f0d2820bacabe6dc9f562785b74af24c5e468
+    1.7.23: 8a0de43d9313aef2ebdccc0ffa49461a4a28139a2c0ef104c3c847f6f37c8119
+    1.7.24: 1a94f15139f37633f39e24f08a4071f4533b285df3cbee6478972d26147bcaef
+    1.7.25: 02990fa281c0a2c4b073c6d2415d264b682bd693aa7d86c5d8eb4b86d684a18c
+    1.7.26: fdf1fb17086b62fc861103da4e3fda3d79bc543b42d2acef5d07e76b13d35d19
+    1.7.27: 5b038fb22ab5dbb1ce57dd3d8f102460cd8619ff2afc78870837b06e8c4e840a
+    1.7.28: 7a8c262deb63becc877e82d23749e4f99f4a17e8e660f9b8c257ca87a5c056b6
+    2.0.0: 6f8da716941f7e89315cefaa6e5a8f1ff10b323ff46611313c455df7ab1ebee1
+    2.0.1: 85061a5ce1b306292d5a64f85d5cd3aff93d0982737a1069d370dd6cb7bbfd09
+    2.0.2: 9bd5b6a1bdf505d520d9a329c520258ed0a17faa9fe3db12712ee858ad59aae3
+    2.0.3: ac70856f1d8bd3aa9ca5d62db5516b86dfa0f934c1fd1d1c5fa4422dd12ba45e
+    2.0.4: e1c64c5fd60ecd555e750744eaef150b6f78d7f750da5c08c52825aa6b791737
+    2.0.5: 88ab31f3e78e4d2fa12dcb933032122d11d441c83b79a89c6c8076f871e50df8
+    2.0.6: a545471a67b8508a3c58ad01a4e6bb2921ace1e59e00a0a2d2e784c1f4fa8caa
+    2.1.0: 0e5359e957b66b679be807563a543c7416e305e3aafcf56bad90ef87a917014d
+    2.1.1: 918e88fd393c28c89424e6535df0546ca36c1dfa7d8a5d685dee70b449380a9b
+    2.1.2: 87c18b2686f38ee6f738492d04fc849f80567b7849d0710ee9d19fac3454adc4
+    2.1.3: 436cc160c33b37ec25b89fb5c72fc879ab2b3416df5d7af240c3e9c2f4065d3c
+    2.1.4: 316d510a0428276d931023f72c09fdff1a6ba81d6cc36f31805fea6a3c88f515
   arm64:
     1.7.17: 8d9749985796a208e860afe331ec77cb485566104e5cc7c0b5e9e82ec7681969
+    1.7.18: e80ce87b469af03b3bdcf68b95f0f4a303787ae247581bcd42f04acf1ad4c24d
+    1.7.19: 1839e6f7cd7c62d9df3ef3deac3f404cdd5cd47bbdf8acfeb0b0f3776eb20002
+    1.7.20: cf80cd305f7d1c23aaf0c57bc1c1e37089cad9130d533db6fe968cdebd16c759
+    1.7.21: 7b6b67d998eb86856d23df5d57269c054539072bbb27677975cf78269b2c5c10
+    1.7.22: 48d0a8461ae829b12b07c3663b14b70287d0607a0792719c51b4e4dd700b02ce
+    1.7.23: 6a66b5e63a5e88ff7eeb478ccaca9083d44e51e1d7261ae183fe5951a6226ccd
+    1.7.24: 420406d2b34ebb422ab3755fbeede59bf3bfcfccf5cfa584b558c93769d99064
+    1.7.25: e9201d478e4c931496344b779eb6cb40ce5084ec08c8fff159a02cabb0c6b9bf
+    1.7.26: adea067914e678ac37d5091ead66f1e36e5cced4d395bbd2be60772495e09eff
+    1.7.27: 3f03ea60c7dacddf890be3ab18f7ef859d9d104b19627f52038d7984361912bc
+    1.7.28: 97457594ff8549cb82d664306593cafd3d2c781c706f9fffed885a46d8919bec
+    2.0.0: 2a00b1553f38aa9e716d61316b661961c2fbfbb7aad7bd73b377be5725ecc0f1
```

Comment:
I think before go live we need implement this for v2: https://github.com/kubernetes-sigs/image-builder/commit/87766222d49690778afefa0be998d588b9229c0e